content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md

```diff
@@ -163,7 +163,7 @@ Note that the kubeadm CLI command never touches this drop-in file.
 
 This configuration file installed by the `kubeadm`
 [package](https://github.com/kubernetes/release/blob/cd53840/cmd/krel/templates/latest/kubeadm/10-kubeadm.conf) is written to
-`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and is used by systemd.
+`/lib/systemd/system/kubelet.service.d/10-kubeadm.conf` and is used by systemd.
 It augments the basic
 [`kubelet.service`](https://github.com/kubernetes/release/blob/cd53840/cmd/krel/templates/latest/kubelet/kubelet.service):
 
```

Comment:
kubeadm is putting the file in the right place, I think. We could use a [generator](https://www.freedesktop.org/software/systemd/man/latest/systemd.generator.html) if we wanted to, but so far we don't.


I'd provide a hint:
```suggestion
If you want to override that further, you can make a directory `/etc/systemd/system/kubelet.service.d/`
(not `/lib/systemd/system/kubelet.service.d/`) and put your own customizations
into a file there.
For example, you might add a new local file `/etc/systemd/system/kubelet.service.d/local-overrides.conf`
to override the unit settings configured by `kubeadm`.

Here is what you are likely to find in `/lib/systemd/system/kubelet.service.d/10-kubeadm.conf`:
```

Reply:
i think that might be too advanced for 95%+ of users.

Reply:
ok, +1 to include the info.
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/kubelet-integration/#the-kubelet-drop-in-file-for-systemd is currently misleading users to modify the package file, which is not ideal.